packages/ember-rdfa-editor/package.json

```diff
@@ -0,0 +1,421 @@
+      "default": "./dist/plugins/image/index.js"
+    },
+    "./plugins/imported-resources": {
+      "types": "./declarations/plugins/imported-resources/index.d.ts",
+      "default": "./dist/plugins/imported-resources/index.js"
+    },
+    "./plugins/invisibles": {
+      "types": "./declarations/plugins/invisibles/index.d.ts",
+      "default": "./dist/plugins/invisibles/index.js"
+    },
+    "./plugins/last-key-pressed": {
+      "types": "./declarations/plugins/last-key-pressed/index.d.ts",
+      "default": "./dist/plugins/last-key-pressed/index.js"
+    },
+    "./plugins/link": {
+      "types": "./declarations/plugins/link/index.d.ts",
+      "default": "./dist/plugins/link/index.js"
+    },
+    "./plugins/list": {
+      "types": "./declarations/plugins/list/index.d.ts",
+      "default": "./dist/plugins/list/index.js"
+    },
+    "./plugins/list/input_rules": {
+      "types": "./declarations/plugins/list/input_rules/index.d.ts",
+      "default": "./dist/plugins/list/input_rules/index.js"
+    },
+    "./plugins/placeholder": {
+      "types": "./declarations/plugins/placeholder/index.d.ts",
+      "default": "./dist/plugins/placeholder/index.js"
+    },
+    "./plugins/rdfa-info": {
+      "types": "./declarations/plugins/rdfa-info/index.d.ts",
+      "default": "./dist/plugins/rdfa-info/index.js"
+    },
+    "./plugins/remove-properties-of-deleted-nodes": {
+      "types": "./declarations/plugins/remove-properties-of-deleted-nodes/index.d.ts",
+      "default": "./dist/plugins/remove-properties-of-deleted-nodes/index.js"
+    },
+    "./plugins/table": {
+      "types": "./declarations/plugins/table/index.d.ts",
+      "default": "./dist/plugins/table/index.js"
+    },
+    "./plugins/text-style": {
+      "types": "./declarations/plugins/text-style/index.d.ts",
+      "default": "./dist/plugins/text-style/index.js"
+    },
+    "./plugins/_private/editable-node": {
+      "types": "./declarations/plugins/_private/editable-node/index.d.ts",
+      "default": "./dist/plugins/_private/editable-node/index.js"
+    },
+    "./components/_private/rdfa-editor": {
+      "types": "./declarations/components/_private/rdfa-editor/index.d.ts",
+      "default": "./dist/components/_private/rdfa-editor/index.js"
+    },
+    "./components/_private/attribute-editor": {
+      "types": "./declarations/components/_private/attribute-editor/index.d.ts",
+      "default": "./dist/components/_private/attribute-editor/index.js"
+    },
+    "./components/_private/debug-info": {
+      "types": "./declarations/components/_private/debug-info/index.d.ts",
+      "default": "./dist/components/_private/debug-info/index.js"
+    },
+    "./vendor/*": "./vendor/*",
+    "./_index.scss": "./_index.scss",
+    "./ember-rdfa-editor.scss": "./ember-rdfa-editor.scss",
+    "./*": {
+      "types": "./declarations/*.d.ts",
+      "default": "./dist/*.js"
+    }
+  },
+  "imports": {
+    "#root": null,
+    "#root/index.ts": null,
```

Comment:
Could we map any `index.ts` file to `null` here?

Reply:
I suppose we could, but we'd have to explicitly list them all, like in the exports field
and it still doesn't prevent relative import usage, so it doesn't _really_ provide safety. I just wanted `#root` itself to be less attractive of an import with this rule
I think for the other indexes it has less added value to exclude them here

Reply:
Ok yeah that makes sense!